packages/qwik-city/adapters/netlify-edge/vite/index.ts

```diff
@@ -49,6 +49,7 @@ export function netlifyEdgeAdapter(opts: NetlifyEdgeAdapterOptions = {}): any {
               path: basePathname + '*',
               function: 'entry.netlify-edge',
               cache: 'manual',
+              excludedPath: Array.isArray(opts.excludedPath) ? opts.excludedPath : [],
```

Comment:
I don't think we can use arrays on excludedPath field, only string:

`7:53:39 AM:   2. Edge Functions bundling                                    
7:53:39 AM: ────────────────────────────────────────────────────────────────
7:53:39 AM: ​
7:53:39 AM: Packaging Edge Functions from .netlify/edge-functions directory:
7:53:39 AM:  - entry.netlify-edge
7:53:39 AM: ​
7:53:39 AM:   Internal error during "Edge Functions bundling"               
7:53:39 AM: ────────────────────────────────────────────────────────────────
7:53:39 AM: ​
7:53:39 AM:   Error message
7:53:39 AM:   TypeError: Expected a string
7:53:39 AM: ​
7:53:39 AM:   Error location
7:53:39 AM:   During Edge Functions bundling
7:53:39 AM:       at module.exports (/opt/buildhome/node-deps/node_modules/@netlify/edge-bundler/node_modules/glob-to-regexp/index.js:3:11)
7:53:39 AM:       at pathToRegularExpression (file:///opt/buildhome/node-deps/node_modules/@netlify/edge-bundler/dist/node/manifest.js:94:31)
7:53:39 AM:       at getExcludedRegularExpression (file:///opt/buildhome/node-deps/node_modules/@netlify/edge-bundler/dist/node/manifest.js:132:16)
7:53:39 AM:       at file:///opt/buildhome/node-deps/node_modules/@netlify/edge-bundler/dist/node/manifest.js:65:33
7:53:39 AM:       at Array.forEach (<anonymous>)
7:53:39 AM:       at generateManifest (file:///opt/buildhome/node-deps/node_modules/@netlify/edge-bundler/dist/node/manifest.js:55:18)
7:53:39 AM:       at writeManifest (file:///opt/buildhome/node-deps/node_modules/@netlify/edge-bundler/dist/node/manifest.js:136:22)
7:53:39 AM:       at bundle (file:///opt/buildhome/node-deps/node_modules/@netlify/edge-bundler/dist/node/bundler.js:78:28)
7:53:39 AM:       at processTicksAndRejections (node:internal/process/task_queues:96:5)
7:53:39 AM:       at async coreStep (file:///opt/buildhome/node-deps/node_modules/@netlify/build/lib/plugins_core/edge_functions/index.js:35:30)`

Reply:
Thanks for the heads up, yeah I was looking at the internal types of it which convert it to a string array. Should be fixed now